pom.xml

```diff
@@ -647,7 +647,7 @@
             <!-- Generate API docs using Doclava for the developer site. -->
             <artifactId>maven-javadoc-plugin</artifactId>
             <!-- Note: version 3.x.x uses additionalOption instead of additionalparam. -->
-            <version>3.1.1</version>
+            <version>3.2.0</version>
```

Comment:
Unrelated to this PR, but you might want to change

```
<additionalOption>/java/reference/</additionalOption>
```

below to be another parameter to make it easier to change to /java/docs/reference later, without having to do another release of java-shared-config.